system-addon/test/unit/lib/UserDomainAffinityProvider.test.js

```diff
@@ -50,7 +49,6 @@ describe("User Domain Affinity Provider", () => {
       }
     };
 
-    ({UserDomainAffinityProvider} = injector());
```

Comment:
gah, I guess I cant remove this one?

```
Firefox 58.0.0 (Mac OS X 10.13.0) ERROR
  TypeError: Cc['@mozilla.org/browser/nav-history-service;1'] is undefined
  at webpack:///system-addon/lib/UserDomainAffinityProvider.jsm:9:0 <- system-addon/test/unit/unit-entry.js:16334
```

Reply:
Or I guess I could add `Cc['@mozilla.org/browser/nav-history-service;1']` to unit-entry.js and it would work. hmm. The only the injector is doing here is delaying the actual import.

Reply:
Turns out it was more complicated ;) I have it fixed locally